supplementary/devices/taoshan.rst

```diff
@@ -0,0 +1,69 @@
+
+Taoshan - Sony Xperia L
+===========================
+
+The device is quite weak to run Halium. But it should be possible.
+
+Status
+------
+
+Halium
+^^^^^^
+A port for Halium 7.1 is being worked on by `Aayush Gupta <https://github.com/jonnius>`_. Halium 7.1 builds, but does not install using halium-install. Using halium-install from JBB it fails to copy the rootfs.img to /data.
```

Comment:
The problems of the official script are caused by the twrp busybox and the problems of @JBBgameich  script are failing because of adb errors